<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="GLWidget" name="glWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>642</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>1080</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QGroupBox" name="gb_district1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>200</width>
         <height>180</height>
        </rect>
       </property>
       <property name="title">
        <string>District 1</string>
       </property>
       <widget class="QLabel" name="lbl_d1MinHeight">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>160</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Min. Height:</string>
        </property>
       </widget>
       <widget class="QSlider" name="slider_d1MinHeight">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>160</width>
          <height>19</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_d1MaxHeight">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>160</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Max. Height:</string>
        </property>
       </widget>
       <widget class="QSlider" name="slider_d1MaxHeight">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>160</width>
          <height>19</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_d1BlockSize">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>160</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Block Size:</string>
        </property>
       </widget>
       <widget class="QSlider" name="slider_d1BlockSize">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>160</width>
          <height>19</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="gb_district2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>200</width>
         <height>180</height>
        </rect>
       </property>
       <property name="title">
        <string>District 2</string>
       </property>
       <widget class="QLabel" name="lbl_d2MinHeight">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>160</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Min. Height:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_d2MaxHeight">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>160</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Max. Height:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_d2BlockSize">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>160</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Block Size:</string>
        </property>
       </widget>
       <widget class="QSlider" name="slider_d2MinHeight">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>160</width>
          <height>19</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <widget class="QSlider" name="slider_d2MaxHeight">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>160</width>
          <height>19</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <widget class="QSlider" name="slider_d2BlockSize">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>160</width>
          <height>19</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="gb_district3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>410</y>
         <width>200</width>
         <height>180</height>
        </rect>
       </property>
       <property name="title">
        <string>District 3</string>
       </property>
       <widget class="QLabel" name="lbl_d3MinHeight">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>160</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Min. Height:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_d3MaxHeight">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>160</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Max. Height:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_d3BlockSize">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>160</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Block Size:</string>
        </property>
       </widget>
       <widget class="QSlider" name="slider_d3MinHeight">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>160</width>
          <height>19</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <widget class="QSlider" name="slider_d3MaxHeight">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>160</width>
          <height>19</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <widget class="QSlider" name="slider_d3BlockSize">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>160</width>
          <height>19</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="gb_seed">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>50</height>
        </rect>
       </property>
       <property name="title">
        <string>Seed</string>
       </property>
       <widget class="QLineEdit" name="txt_seed">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>160</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="btn_generate">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>600</y>
         <width>160</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
